app/models/maths_and_physics.rb

```diff
@@ -18,11 +18,11 @@ def notify_reply_to_id
   end
 
   def feedback_url
-    "https://docs.google.com/forms/d/e/1FAIpQLSeJcp50-eA5H_twIFKTUX8Z1ATDnj63wZaSlcnBCN-idJ7Ztg/viewform?usp=sf_link"
+    "https://docs.google.com/forms/d/e/1FAIpQLSfwPUxmNHqSnQ6RJ-0nedu5F2FRibBF5UIJ_EciTPcWQg581A/viewform"
   end
 
   def done_page_url
```

Comment:
This method was only introduced because we wanted to have different behaviour between the two policies. Now that we've settled on the decision that there will only be a single done page for both/all policies, rather than changing this method we can go further and remove all the code that was introduced to split the logic: https://github.com/DFE-Digital/dfe-teachers-payment-service/pull/647/files

Reply:
I did wonder about that, given that there's likely to be other policies coming down the pipe, but there's no reason why this functionality can't be put back if/when that happens.

Reply:
My understanding of your reply is that you would remove the logic for having an optional done page, @pezholio. Is that correct? It seems to still be there.

Reply:
Ah, I thought I'd fixed that

Reply:
This code is still here.

Reply:
Ach. Must be a bad rebase. I'll try again 🙄